src/client/src/rt-theme/tools.ts

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash'
```

Comment:
Hard to follow! Also things like this should be tested

Reply:
Yes, but their behavior is brilliant — this is an oldie but a goodie, wrote it two years ago. But still, not using, so I'm removing it for now.